src/tasks/daily/daily_trade_mixin.py

```
@@ -327,7 +327,12 @@ def buy_sell(self, target_areas=None, keep_area_context=False, after_buy=None):
             if not (buy_price and sell_price):
                 self.log_info("未找到买入价或卖出价")
                 continue
-            if self.config.get("只买不卖", False):
+            buy_only = self.config.get("只买不卖", False)
+            if self.input_mode() == "background":
+                if not buy_only:
+                    self.log_info("后台模式下强制只买不卖，跳过卖出")
```

Ruff (0.16.2)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)